MEMBER_EXPECTATIONS.md

```diff
@@ -1,7 +1,7 @@
 All participants in the OpenJS Foundation projects and groups must follow the
 [Code of Conduct](CODE_OF_CONDUCT.md). There are further expectations for
 members of leadership teams including the
-([CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
+([CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md)).
```

Comment:
I think the correct thing to do is to remove the opening parenthesis rather than adding a closed one. It should be `including the CPC` and not `including the (CPC)`.

Reply:
Ha, indeed. Should've read the full sentence first :). Fixed!